### PR TITLE
Gumstix-pixhawk-issue466

### PR DESCRIPTION
**Pull request by ExplodingLemur**
Add Gumstix Pixhawk board per https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/466

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -65,9 +65,11 @@ GEM
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
+    webrick (1.8.1)
 
 PLATFORMS
   ruby
+  x64-mingw-ucrt
```
Comment (geerlingguy) [inline Gemfile.lock:72]: Can you explain what webrick and this new platform would help with? Or were those just changes for your local development environment?

File: _boards_cm/gumstix-pixhawk-fmuv6u-cm4.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: board
+title: "Gumstix Pixhawk FMUv6U + Pi CM4"
+short_description: This board incorporates a CM4, a Pixhawk FMUv6U compliant flight controller, and a Google Coral TPU.
+status: production
+picture: "/images/board-gumstix-pixhawk-fmuv6-cm4.jpg"
+github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/466"
+link: https://www.gumstix.com/manufacturer/cm4-pixhawk-fmuv6.html
+videos: [] IF NO VIDEOS
```
Comment (geerlingguy) [inline _boards_cm/gumstix-pixhawk-fmuv6u-cm4.md:9]: This should just be `videos: []` in this case.

**Review by geerlingguy — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (ExplodingLemur) [reply Gemfile.lock:72]: webrick and x64-mingw-ucrt were to make jekyll execute on my machine (yep, Windows). I can pull that change out of the PR.

File: _boards_cm/gumstix-pixhawk-fmuv6u-cm4.md (hunk 2 above)
Comment (ExplodingLemur) [reply _boards_cm/gumstix-pixhawk-fmuv6u-cm4.md:9]: Whoops. I made a pseudo-template for myself with some notes and failed to edit that out. Will re-submit.

**Pull request closed without merging**